src/web/mjs/connectors/BacaMangaOrg.mjs

```diff
@@ -1,13 +1,13 @@
 import WordPressMangastream from './templates/WordPressMangastream.mjs';
 
-export default class BacaMangaOrg extends WordPressMangastream {
+export default class MangaTale extends WordPressMangastream {
```

Comment:
I am not fond of changing class name but there is probably no consequence.

Reply:
file should have the same name as the class and ID. 
changing the ID will lose the user bookmarks related to it as it would be considered as a new connector.

for me, it's a new connector or you let it like that.

Reply:
Oh yeah i forgot. We already got the case. Old id is kept because changing it causes problems. I guess class must stays the same too, to be matching filename.